bartop-ui/src/components/DrinksList/DrinksList.js

```diff
@@ -0,0 +1,40 @@
+import React from 'react';
+import { Query } from 'react-apollo';
+import gql from 'graphql-tag';
+import styled from 'styled-components';
+
+const Container = styled.div`
+  display: block;
+  width: 80%;
+`;
+
+const DrinksList = () => (
+  <Query
+    query={gql`
+      {
+        listDrinks {
+          id
+          name
+          description
+        }
+      }
+    `}
+  >
```

Comment:
Idk where it would be, but we should be setting the authentication header when making a query